src/sentry/lang/javascript/processor.py

```diff
@@ -760,6 +765,12 @@ def process_frame(self, frame, stacktrace_info, idx):
                     # We want to explicitly generate a webpack module name
                     new_frame['module'] = generate_module(filename)
 
+                if is_react_native:
+                    if 'node_modules' in filename:
```

Comment:
we should probably regex match on `\bnode_modules/` here to avoid false positives.